cmd/dra-example-controller/gpu.go

```diff
@@ -110,15 +110,13 @@ func (g *gpudriver) allocate(crd *nascrd.NodeAllocationState, pod *corev1.Pod, g
 	available := make(map[string]*nascrd.AllocatableGpu)
 
 	for _, device := range crd.Spec.AllocatableDevices {
-		switch device.Type() {
-		case nascrd.GpuDeviceType:
+		if device.Type() == nascrd.GpuDeviceType {
 			available[device.Gpu.UUID] = device.Gpu
 		}
 	}
 
 	for _, allocation := range crd.Spec.AllocatedClaims {
-		switch allocation.Type() {
-		case nascrd.GpuDeviceType:
+		if allocation.Type() == nascrd.GpuDeviceType {
```

Comment:
Let's keep the switch here.

Reply:
What would the switch switch? We cannot even return an error from default in here, we just trust that there's only one type; so the default action will have to be empty like above, would that be fine ?

Reply:
See above.

Reply:
Done.